examples/sockend/sockend.js

```diff
@@ -1,8 +1,17 @@
-var cote = require('cote'),
-    app = require('http').createServer(handler),
+let cote;
+try {
+    cote = require('cote');
+} catch (e) {
+    cote = require('../../');
+}
+let app = require('http').createServer(handler),
     io = require('socket.io').listen(app),
     fs = require('fs');
 
+io.on('connection', (socket)=>{
```

Comment:
Missing spaces here as well.

Reply:
same here...

Reply:
it should be `io.on('connection', (socket) => {`